docs/governance/policies.md

```diff
@@ -0,0 +1,130 @@
+- The original installation (or a substantial part thereof) was found to be broken. Not that this does _not_ apply to small bugs.
+- Mistakes in the original installation prevent it from being used as a dependency for other software
+
+Additionally, software may be rebuilt for other reasons if there is sufficient consensus within the EESSI Community that this is beneficial.
+
+In all cases, the value of rebuilding the software will be carefully weighed against the value of reproducibility.
+
+### 2.3 Deprecation Policy
+
+#### 2.3.1 Deprecation of EESSI software stack versions
+At any given time, we aim to have at least two EESSI software stack versions in production. When a new version of the EESSI software stack is introduced _and_ sufficiently populated, the oldest stack will be deprecated. A clear runtime warning will be printed when an end-user initializes a deprecated EESSI software stack.
+
+#### 2.3.2 Deprecation of individual software installations
+
+Installations of individual software packages in a (non-deprecated) EESSI software stack may be deprecated if they are no longer maintainable or functional. A clear runtime warning will be printed when an end-user tries to load a module for a deprecated software installation.
+
+### 2.4 Removal Policy
+
+#### 2.4.1 Removal of EESSI software stack versions
+
+Since CernVM-FS Stratum 1 servers have to fully mirror the repository, old EESSI software stack versions will have to be removed to keep the storage footprint within reasonable limits. This will only be done after:
+- The EESSI software stack version has been deprecated for at least 1 year
+- The EESSI software stack version has been archived in _some_ form in which it can still be used by end-users (e.g. as a tarball, an exported CVMFS repository using CernVM-FS Shrinkwrap, or some other form).
+
+A clear runtime error will be printed when an end-user tries to initialize a removed EESSI software stack.
+
+#### 2.4.2 Removal of individual software installations
+
+An individual software installation in EESSI may be removed if:
+- It violates licensing terms
+- It poses a security risk
+- It is no longer maintainable or functional.
+
+A clear runtime error will be printed when an end-user tries to load a module for a removed software installation.
+
+### 2.5 Optimization Policy
+
+Optimization for performance is strongly encouraged. For compiled software, this means we aim to have the compiler optimize for the supported hardware targets.
+
+---
+
+## 3. Contribution Policies
+
+### 3.1 Software Acceptance
+
+- Contributions of new software are accepted as long as they meet the [Contribution Policy](https://www.eessi.io/docs/adding_software/contribution_policy/).
+- The default stance is: **"Yes, unless we can't."**
+
+### 3.2 Non-Software Contributions
+
+- Contributions to build logic, test suites, or infrastructure must be testable and documented.
```

Comment:
I would tone this down to "should", since we currently often make changes that are not documented:

```suggestion
- Contributions to build logic, test suites, or infrastructure should be testable and documented.